wots.c

```diff
@@ -129,6 +203,79 @@ void wots_pkgen(const xmss_params *params,
  * Takes a n-byte message and the 32-byte seed for the private key to compute a
  * signature that is placed at 'sig'.
  */
+typedef struct wots_sign_args
+{
```

Comment:
Can you comment the members of this struct?

Reply:
fix in https://github.com/kj3moraes/further-testing-xmss/pull/5/commits/46648c53b9aa817fef5cde8cdf7c1d2004a92b05

Reply:
I don't see comments added.

Reply:
Please check in this commit: https://github.com/kj3moraes/further-testing-xmss/pull/5/files#diff-8abae1a1630bbc61fc3689c457ec80afbcb32ccae984fcda0db7a26b71ffce0bR121

Reply:
fix in this commit: https://github.com/kj3moraes/further-testing-xmss/pull/5/commits/fc925eb997a4d8ce9827f174f69eedde9c6293cb